src/searchModal/scss/searchModal.scss

```diff
@@ -0,0 +1,75 @@
+.search-modal {
+    height: 100%;
+    padding: 0px;
+    .navi-container {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        padding: 40px 20px 20px 20px;
+        .search-bar-container {
+            position: relative;
+            .icon-find {
+                position: absolute;
+                top: 5px;
+                left: 5px;
+            }
+            input {
+                width: 100%;
+                padding-left: 25px;
+            }
+        }
+        .buttons-container {
+            display: flex;
+            justify-content: space-around;
+            button {
+                width: 45%;
+            }
+            .btn-transparent {
+                background-color: transparent;
+                border: 1px solid black;
+                color: black;
+                text-shadow: none;
+            }
+        }
+    }
+    .content-container {
+        .no-datatable-container {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            .no-datatable-icon {
+                color: lightgray;
```

Comment:
Since for different clients we're using different themes, we need to use a global css variables 🤔
Could you include on the top `@import "inc/bootstrap";` and pick the proper color name from https://github.com/oat-sa/tao-core-ui-fe/blob/master/scss/inc/_colors.scss ?